source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -198,7 +208,6 @@ class CreepVote extends Component {
           flowText={getText(
             `Now that you’re on a roll, why not join Mozilla? We’re not creepy (we promise). We actually fight back against creepy. And we need more people like you.`
           )}
-          csrfToken={this.props.joinUsCSRF}
```

Comment:
this.. is nothing, and was nothing. the JoinUs component doesn't use a CSRT token.